python/hail/methods/misc.py

```diff
@@ -82,6 +82,10 @@ def maximal_independent_set(i, j, keep=True, tie_breaker=None) -> Table:
 
     For example, the usual ordering on the integers is defined by: ``l - r``.
 
+    The `tie_breaker` function must satisfy the following properties:
+    ``tie_breaker(l, r) == -tie_breaker(r, l)`` OR
+    ``tie_breaker(l, r) == 0 and tie_breaker(r, l) == 0``.
```

Comment:
The mathematician in me wants to point out that the second line is unnecessary, but if you think it would be confusing to users to remove it, that's fine.